dietpi/func/dietpi-banner

```diff
@@ -327,19 +350,35 @@ $GREEN_LINE"
 		do
 			aENABLED[$i]=1
 
-			# Custom entry
-			(( $i == 10 )) || continue
-
-			[[ -f $FP_CUSTOM ]] && read -r G_WHIP_DEFAULT_ITEM < "$FP_CUSTOM" || G_WHIP_DEFAULT_ITEM='echo '\''Hello World!'\'
-			G_WHIP_INPUTBOX 'You have chosen to show a custom entry in the banner.
+			if (( $i == 10 ))
+			then
+				# Custom entry
+				[[ -f $FP_CUSTOM ]] && read -r G_WHIP_DEFAULT_ITEM < "$FP_CUSTOM" || G_WHIP_DEFAULT_ITEM='echo '\''Hello World!'\'
+				G_WHIP_INPUTBOX 'You have chosen to show a custom entry in the banner.
 Please enter the desired command here.\n
 NB: It is executed as bash script, so it needs to be in bash compatible syntax.
     For multi-line or non-bash scripts, keep it separate and only add the script call here.' || continue
 
-			echo "$G_WHIP_RETURNED_VALUE" > "$FP_CUSTOM"
+				echo "$G_WHIP_RETURNED_VALUE" > "$FP_CUSTOM"
+
+				G_WHIP_DEFAULT_ITEM=${aDESCRIPTION[10]}
+				G_WHIP_INPUTBOX 'Please enter a meaningful name to be shown in front of your custom command output:' && aDESCRIPTION[10]=$G_WHIP_RETURNED_VALUE
 
-			G_WHIP_DEFAULT_ITEM=${aDESCRIPTION[10]}
-			G_WHIP_INPUTBOX 'Please enter a meaningful name to be shown in front of your custom command output:' && aDESCRIPTION[10]=$G_WHIP_RETURNED_VALUE
+			elif (( $i == 19 ))
+			then
+				# Banner word wrap options
+				G_WHIP_MENU_ARRAY=(
+					'min' 'Indent to a fixed minimum offset'
+					'dash' 'Indent to the green dash character'
+					'colon' 'Indent to the green colon character'
+					'dashcolon' 'Indent first to Colon then Dash'
```

Comment:
This does not seem to work yet. It either aligns to colon, the last one in the `make_whitespace_offset` function, or to `INDENT_MIN`, the fallback in main awk block. But checking this in awk script.

Reply:
oh yep, I could have been a bit clearer with the definitions there.

`INDENT_MIN` was meant as the absolute fallback indentation strategy if no colon or dash was discovered in the line. Some users might want this min to be wider than the dash indent, but yes probably most will not.

> So basically INDENT_MIN variable becomes INDENT_FIXED, and where it was used as fallback for the other options, we use a fixed 1?

Sounds good!